ddtrace/internal/_encoding.pyx

```diff
@@ -273,37 +273,60 @@ cdef class Packer(object):
 
         return ret
 
-    cdef int _pack(self, object o) except -1:
+    cdef _pack_trace(self, object trace):
```

Comment:
```suggestion
    cdef _pack_trace(self, list trace):
```

? since we get it as a `list trace` in `pack_trace`

Reply:
or can we not do `list trace` since this is a `cdef`?

Reply:
I assumed `cdef` functions don't accept `list` parameters, but it seems to be fine when I applied those changes. I'll read some documentation to verify.